src/app/space/analyze/analyze-overview/analyze-overview.spec.ts

```diff
@@ -0,0 +1,228 @@
+import { AnalyzeOverviewComponent } from './analyze-overview.component';
+
+import {
+  Component,
+  DebugElement,
+  Input
+} from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { Broadcaster } from 'ngx-base';
+import { BsModalService } from 'ngx-bootstrap/modal';
+import { Context, Contexts, Space } from 'ngx-fabric8-wit';
+import { AuthenticationService, User, UserService } from 'ngx-login-client';
+import { ConnectableObservable, Subscription } from 'rxjs';
+import { Observable, Subject } from 'rxjs';
+import { createMock } from 'testing/mock';
+import {
+  initContext,
+  TestContext
+} from 'testing/test-context';
+
+@Component({
+  template: '<alm-analyzeOverview></alm-analyzeOverview>'
+})
+class HostComponent { }
+
+@Component({
+  selector: 'fabric8-edit-space-description-widget',
+  template: ''
+})
+class FakeFabric8EditSpaceDescriptionWidget {
+}
+
+@Component({
+  selector: 'fabric8-work-item-widget',
+  template: ''
+})
+class FakeFabric8WorkItemWidget {
+}
+
+@Component({
+  selector: 'fabric8-pipelines-widget',
+  template: ''
+})
+class FakeFabric8PipelinesWidget {
+}
+
+@Component({
+  selector: 'fabric8-add-codebase-widget',
+  template: ''
+})
+class FakeFabric8AddCodebaseWidget {
+}
+
+@Component({
+  selector: 'fabric8-analytical-report-widget',
+  template: ''
+})
+class FakeFabric8AnalyticalReportWidget {
+}
+
+@Component({
+  selector: 'fabric8-create-work-item-widget',
+  template: ''
+})
+class FakeFabric8CreateWorkItemWidget {
+}
+
+@Component({
+  selector: 'fabric8-environment-widget',
+  template: ''
+})
+class FakeFabric8EnvironmentWidget {
+}
+
+@Component({
+  selector: 'f8-feature-toggle',
+  template: ''
+})
+class FakeFabric8FeatureToggle {
+}
+
+@Component({
+  selector: 'flow-selector',
+  template: ''
+})
+class FlowSelector {
+  @Input() space: Space;
+}
+
+@Component({
+  selector: 'import-wizard',
+  template: ''
+})
+class ImportWizard {
+}
+
+@Component({
+  selector: 'quickstart-wizard',
+  template: ''
+})
+class QuickstartWizard {
+}
```

Comment:
I think you can probably get away with not mocking and declaring a lot of these fake components using `NO_ERRORS_SCHEMA` (https://angular.io/api/core/NO_ERRORS_SCHEMA). Please see which you can get away with not mocking here to reduce the amount of boilerplate in this file.